build.py

```diff
@@ -0,0 +1,58 @@
+from subprocess import check_call
+
+BUILD_DIR = "_build"
+
+check_call(
```

Comment:
Given that doc2dash is installed via requirements, is it OK to assume that it is available globally and I can just use it?
I mean `Makefile` allows overriding path to `sphinx-build` is this needed for dash too?

In other words I suggest removing call to `sphinx-build` from this file and add `Makefile`target which runs this python script.

Does it sound OK?

Reply:
Yes, yes, I think so (the only requirement is that doc2dash is something that installs nicely on most platforms). So it can be assumed that it is there and use it.

Makefile targets sounds perfect!